packages/visualizations/addon/templates/components/cell-renderers/dimension.hbs

```diff
@@ -1,6 +1,10 @@
 {{!-- Copyright 2017, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
 {{#with (get data (concat column.field.dimension '|desc' )) as |description|}}
-    <span title={{description}}>{{description}}</span>
+    {{#with (get data (concat column.field.dimension '|id' )) as |id|}}
+        <span title={{id}}>{{description}}</span>
```

Comment:
Can we do `{{description}} ({{id}})` in the title? I think we follow the same format elsewhere.

Reply:
we do, but only when desc and ID are far from each other.  this isn't the case here.